lib/actions/app.coffee

```diff
@@ -44,7 +45,13 @@ exports.create =
 					return callback(new Error('You already have an application with that name!'))
 
 				return callback(null, options.type) if options.type?
-				visuals.patterns.selectDeviceType(callback)
+				resin.models.device.getSupportedDeviceTypes().then (supportedDeviceTypes) ->
+					form.ask
+						message: 'Device Type'
+						name: 'deviceType'
```

Comment:
`form.ask()` has a default value for a `name` (https://github.com/resin-io/resin-cli-form/blob/master/lib/form.coffee#L79), so it's not necessary here. Can you omit a explicit `name` property in all single `form.ask()` invocations for simplicity?